cloudmarker/test/test_azcloud.py

```diff
@@ -580,3 +584,27 @@ def test_mysql_server_record(self):
                          'azure_mysql_server_id')
         self.assertEqual(records[0]['com']['record_type'],
                          'rdbms')
+
+    def test_postgresql_server_record(self):
+        mock_postgresql_server_dict = {
+            'id': 'azure_postgresql_server_id',
+            'ssl_enforcement': 'Enabled',
+        }
+        mock_postgresql_server = SimpleMock(mock_postgresql_server_dict)
+
+        m = self._MockPostgreSQLManagementClient
+        m().servers.list.return_value = [mock_postgresql_server]
+
+        records = list(azcloud.AzCloud('', '', '').read())
+        records = [
+            r for r in records
+            if r['ext']['record_type'] == 'postgresql_server'
+        ]
+        self.assertEqual(records[0]['ext']['ssl_connection_enabled'],
```

Comment:
I believe this assert would change to `self.assertEqual(records[0]['com']['tls_enforced'], True)` after this pull request is rebased on the latest `master`.